app/containers/Avatar.js

```diff
@@ -1,80 +1,103 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { View } from 'react-native';
-import FastImage from 'react-native-fast-image';
+import React from "react";
+import PropTypes from "prop-types";
+import { View } from "react-native";
+import FastImage from "react-native-fast-image";
 
-const formatUrl = (url, baseUrl, uriSize, avatarAuthURLFragment) => (
-	`${ baseUrl }${ url }?format=png&width=${ uriSize }&height=${ uriSize }${ avatarAuthURLFragment }`
-);
+var date = new Date();
+var timestamp = date.getTime();
 
-const Avatar = React.memo(({
-	text, size, baseUrl, borderRadius, style, avatar, type, children, userId, token
-}) => {
-	const avatarStyle = {
-		width: size,
-		height: size,
-		borderRadius
-	};
+const formatUrl = (url, baseUrl, uriSize, avatarAuthURLFragment) =>
+  `${baseUrl}${url}?format=png&width=${uriSize}&height=${uriSize}${avatarAuthURLFragment}&t=${timestamp}`;
```

Comment:
By using a timestamp on the link, you're breaking the entire avatar cache.
Avatar will start to load properly, but they will be requested from network all the time.

Reply:
I have fixed the lint errors which is caused by Prettier. Please review it.